src/pages/GameV2.js

```diff
@@ -194,6 +200,12 @@ export default class GameV2 extends Component {
     this.gameModel.chat(username, id, message);
   };
 
+  updateSeen = (message) => {
```

Comment:
can we rename this to `updateSeenChatMessage`?

Reply:
done